osmosis/assetlist.json

```diff
@@ -27193,6 +27193,190 @@
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/crypto.png"
+      },
+      "socials": {
+        "website": "https://getcryptomeme.com",
+        "twitter": "https://getcryptomeme"
+      },
+      "keywords": [
+        "meme"
+      ]
+    },
+    {
+      "description": "Bricscoin",
+      "extended_description": "BRICS coin",
+      "denom_units": [
+        {
+          "denom": "ibc/54C7B977B96DFC56D877A7F74D22BD26B5E42CA6171F120F5A36AC82BFD4711A",
+          "exponent": 0,
+          "aliases": [
+            "factory/inj1s9hr5zfz3xrkzchde94hd2d0edjs4q5mrqrz6x/BRICS"
+          ]
+        },
+        {
+          "denom": "BRICS",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/54C7B977B96DFC56D877A7F74D22BD26B5E42CA6171F120F5A36AC82BFD4711A",
+      "name": "Brics coin",
+      "display": "BRICS",
+      "symbol": "BRICS",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "factory/inj1s9hr5zfz3xrkzchde94hd2d0edjs4q5mrqrz6x/BRICS",
+            "channel_id": "channel-8"
+          },
+          "chain": {
+            "channel_id": "channel-122",
+            "path": "transfer/channel-122/factory/inj1s9hr5zfz3xrkzchde94hd2d0edjs4q5mrqrz6x/BRICS"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/brics.png"
+      },
+      "socials": {
+        "website": "https://bricscoin.cash",
+        "twitter": "https://bricscoincash"
+      },
+      "keywords": [
+        "meme"
+      ]
+    },
+    {
+      "description": "Instant Noodles Coin",
+      "extended_description": "Instant Noodles Coin",
+      "denom_units": [
+        {
+          "denom": "ibc/A436BC9498153DC580B56BD27043BBFE08A1754696C05FFF00B2A41EC796D180",
+          "exponent": 0,
+          "aliases": [
+            "factory/inj1s9hr5zfz3xrkzchde94hd2d0edjs4q5mrqrz6x/INC"
+          ]
+        },
+        {
+          "denom": "INC",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/A436BC9498153DC580B56BD27043BBFE08A1754696C05FFF00B2A41EC796D180",
+      "name": "Instant Noodles Coin",
+      "display": "INC",
+      "symbol": "INC",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "factory/inj1s9hr5zfz3xrkzchde94hd2d0edjs4q5mrqrz6x/INC",
+            "channel_id": "channel-8"
+          },
+          "chain": {
+            "channel_id": "channel-122",
+            "path": "transfer/channel-122/factory/inj1s9hr5zfz3xrkzchde94hd2d0edjs4q5mrqrz6x/INC"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/inc.png"
```

Comment:
Please don't upload the images to the /osmosis/images/ directory, since it is duplication. These URLs should instead refer to the /injective/images/ directory instead.